src/VirtoCommerce.DemoSolutionFeaturesModule.Data/Migrations/20201119110433_InitialCustomerSegments.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace VirtoCommerce.DemoSolutionFeaturesModule.Data.Migrations
+{
+    public partial class InitialCustomerSegments : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "DemoCustomerSegments",
+                columns: table => new
+                {
+                    Id = table.Column<string>(maxLength: 128, nullable: false),
+                    CreatedDate = table.Column<DateTime>(nullable: false),
+                    ModifiedDate = table.Column<DateTime>(nullable: true),
+                    CreatedBy = table.Column<string>(maxLength: 64, nullable: true),
+                    ModifiedBy = table.Column<string>(maxLength: 64, nullable: true),
+                    Name = table.Column<string>(maxLength: 128, nullable: false),
+                    Description = table.Column<string>(maxLength: 1024, nullable: false),
+                    IsActive = table.Column<bool>(nullable: false),
+                    StartDate = table.Column<DateTime>(nullable: true),
```

Comment:
StartDate is required, don't it?

Reply:
Shouldn't we add index to this field?

Reply:
No, `StartDate` isn't required, we can leave it empty in blade. The only required field is StoreIds.
Why we need an index?

Reply:
After discussion: leave as is, ask why it set in FromModel in PromotionEntity